WordPress/Classes/ViewRelated/Post/Scheduling/HalfScreenPresentationController.swift

```diff
@@ -0,0 +1,57 @@
+import Foundation
+
+class HalfScreenPresentationController: FancyAlertPresentationController {
+
+    private weak var tapGestureRecognizer: UITapGestureRecognizer?
+
+    override var frameOfPresentedViewInContainerView: CGRect {
+        let height = containerView?.bounds.height ?? 0
+        let width = containerView?.bounds.width ?? 0
+        if traitCollection.verticalSizeClass != .compact {
+            return CGRect(x: 0, y: height/2, width: width, height: height/2)
+        } else {
```

Comment:
I typically like the keep the `else` in case someone removes the early return in the first block. This whole block can just use a `guard` statement, though and revert to the super's implementation.